app/models/manageiq/providers/inventory/persister.rb

```diff
@@ -52,8 +52,16 @@ def inventory_collections_names
     collections.keys
   end
 
+  # Defines how inventory objects will be loaded from the database
+  #
+  # Allowed values are:
+  # * nil - Default strategy, InventoryObjects will be saved and only objects in
+  #         an InventoryCollection can be referenced.  Best used for full refreshes.
+  # * :local_db_find_missing_references - InventoryObjects will be saved and
+  #         lazy_find references will be loaded from the database.  Best used
+  #         for targeted refreshes.
```

Comment:
There are actually two other strategies but they only make sense per-inventory-collection not at the persister level.